Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -351,11 +353,17 @@ private void Update()
             elapsed = 0f;
         }
 
+        if(currentOverlay != "None" && currentLayer != WorldController.Instance.cameraController.CurrentLayer)
+        {
+            Bake();
+            currentLayer = WorldController.Instance.cameraController.CurrentLayer;
+            elapsed = 0f;
+        }
         // TODO: Prettify.
```

Comment:
If this is otherwise mergeable, could a Gatekeeper fix these? As I won't be able to access it until tomorrow. I have *Allow edits from maintainers.* turned on, so I'm assuming it's possible (never had the opportunity to try it myself).